src/homepage/Hero.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { VideoModal } from './VideoModal.jsx';
+import iconSprite from "../common/uswds/img/sprite.svg";
+
+// const [showModal, setShowModal] = useState(false);
+
+// const handleShowModal = () => {
+//   setShowModal(true);
+// };
```

Comment:
If the code is not needed, can we remove this code?

Reply:
Code removed.